### PR TITLE
Split execute into own file

### PR DESCRIPTION
**Pull request by pdxjohnny**
Lets us use gifs in our cli tools

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,9 +166,24 @@ Chances are gif-for-cli was installed in a location not on your `PATH`. This can
...
+```
+
 ## Disclaimer
 
 This is not an officially supported Google product.
 
 [tenor-home]: https://tenor.com/
-[tenor-gif-api]: https://tenor.com/gifapi
\ No newline at end of file
+[tenor-gif-api]: https://tenor.com/gifapi
```
Comment (pdxjohnny) [inline README.md]: vim did something with this newline, i can't figure out how to not change this line

File: tests/test_main.py [hunk 2]
```diff
@@ -18,14 +18,14 @@
 import unittest
```
Comment (SeanHayes) [inline tests/test_main.py]: Could you rename this file to test_execute?

Comment (pdxjohnny) [reply tests/test_main.py]: Got it! I was getting the [errors travis is getting for 3.4](https://travis-ci.org/google/gif-for-cli/jobs/408651285#L538) when I did that locally so I had left it. But now I realize that's [been there](https://travis-ci.org/google/gif-for-cli/jobs/395679764#L539)

**Comment by SeanHayes** [comment]
Thanks!

**Pull request merged**